README.md

```diff
@@ -1,16 +1,30 @@
-# Prerequisites
+# Charity Manager
+
+- [Charity Manager](#charity-manager)
+  - [Prerequisites](#prerequisites)
+  - [Access the code](#access-the-code)
+  - [Run/debug code in IntelliJ](#rundebug-code-in-intellij)
+  - [Deploy and run the code locally as docker instance](#deploy-and-run-the-code-locally-as-docker-instance)
+  - [Code Conventions](#code-conventions)
+    - [Solving tickets](#solving-tickets)
+
+## Prerequisites
 
 Follow the [./PREREQUISITES.md](./PREREQUISITES.md) instructions to configure a local virtual machine with Ubuntu, Docker, IntelliJ.
 
-# Access the code
+---
```

Comment:
`---` not really necessary, as `##` headersalso add a horizontal line (which is below the text, to be fair), when seen on GitHub.

Reply:
Good point! :+1: